src/api/digital-exchange/components.js

```diff
@@ -36,7 +43,26 @@ export const getDEComponentInstall = id => (
   makeRequest({
     uri: `/api/digitalExchange/install/${id}`,
     method: METHODS.GET,
-    mockResponse: COMPONENT_INSTALLATION_IN_PROGRESS,
+    mockResponse: COMPONENT_INSTALLATION_COMPLETED,
```

Comment:
why this change?

Reply:
this was changed because when using mock mode it doesn't really finish the mock installation process when you try to install. I can change it back though but I might as well add a comment to use `COMPONENT_INSTALLATION_COMPLETED` when testing installation in mock mode. would that be okay?

Reply:
ok then, keep your change

Reply:
reverted